android/guava/src/com/google/common/hash/BloomFilter.java

```diff
@@ -214,7 +214,7 @@ public long approximateElementCount() {
 
   /** Returns the number of bits in the underlying bit array. */
   @VisibleForTesting
```

Comment:
now that we've made it public should we remove the annotation? 

The documentation states the following

> Do not use this interface for public or protected declarations: it is a fig leaf for bad design, and it does not prevent anyone from using the declaration---and experience has shown that they will. If the method breaks the encapsulation of its class, then its internal representation will be hard to change. Instead, use RestrictedApiChecker , which enforces fine-grained visibility policies.